src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -44,20 +44,21 @@
         "symfony/process": "~2.8|~3.0",
         "symfony/security-core": "~3.2",
         "symfony/security-csrf": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|~3.0",
+        "symfony/serializer": "~3.3",
         "symfony/translation": "~2.8|~3.0",
         "symfony/templating": "~2.8|~3.0",
         "symfony/validator": "~3.2",
         "symfony/yaml": "~3.2",
-        "symfony/property-info": "~2.8|~3.0",
+        "symfony/property-info": "~3.1",
```

Comment:
Well, then it means that the serializer component should have the proper requirement or the proper conflict in place